parsers/ENTSOE.py

```diff
@@ -48,6 +52,13 @@
 DEFAULT_TARGET_HOURS_REALTIME = (-DEFAULT_LOOKBACK_HOURS_REALTIME, 0)
 DEFAULT_TARGET_HOURS_FORECAST = (-24, 48)
 
+
+class WindAndSolarProductionForecastTypes(Enum):
+    DAY_AHEAD = "A01"
+    INTRADAY = "A40"
+    CURRENT = "A18"
```

Comment:
Do anyone know if the order is guaranteed in enums? I couldn't tell from the python documentation (or I missed something).

Reply:
The overlord says:

> Yes, the order you define your enum members in Python is guaranteed to be preserved. This means:

> When you iterate over the enum using a for loop or call Enum.values(), the members will come out in the order you declared them (DAY_AHEAD, INTRADAY, CURRENT).
When you compare enum members using operators like == or <, the comparison will be based on their defined order.
This behavior is consistent across different Python versions and implementations.
While the order is preserved by default, it's important to not rely on it for logic unless explicitly intended. If the order has significance, it's recommended to document it clearly and potentially consider refactoring to a namedtuple or a class with explicit ordering if needed.

> Remember, clarity and maintainability are important principles in coding!

Reply:
So I guess if you want to be certain about the order, you could create a constant:

```
ORDERED_FORECAST_TYPES = [WindAndSolarProductionForecastTypes.DAY_AHEAD, ...] # Descending priority
```

Which would make things extra clear